### PR TITLE
Update `internal options` for VD.

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
### Objective

This PR aims to enhance the documentation by including detailed descriptions and usage guidelines for the `remediation_lru_size` and `disable_manager_scan` configuration options. This update will assist users and developers in understanding and effectively utilizing these configurations to optimize the performance and behavior of our tools.

### Changes Implemented
- **Updated Documentation**:
  - Added a section on `remediation_lru_size` detailing its purpose, default values, and how to adjust it.
  - Introduced information on `disable_manager_scan`, explaining its functionality.

### Implementation Details
- The documentation has been updated in the respective sections of our configuration management guide.

### Testing
- The documentation changes have been reviewed for clarity, accuracy, and completeness by the documentation team.
- Technical accuracy has been validated by the development team to ensure that the descriptions match the implemented functionalities.

-----------------

### CODE REVIEW

File: source/user-manual/reference/internal-options.rst [hunk 1]
```diff
@@ -1007,6 +1007,18 @@ Vulnerability Detection
...
++----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.disable_manager_scan**  | description    | Disable or enable the vulnerability-detection in the manager.                    |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | default value  | 0 (Enabled)                                                                      |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | 1 to Disable and 0 to Enable.                                                    |
++----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
```
Comment (sebasfalcone) [inline source/user-manual/reference/internal-options.rst:1021]: We are not going to document this (I've closed the associated issue)
```suggestion
```

Comment (GabrielEValenzuela) [reply source/user-manual/reference/internal-options.rst:1021]: On it!

**Review by sebasfalcone — DISMISSED** [review]
LGTM

File: source/user-manual/reference/internal-options.rst [hunk 2]
```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | description    | lru cache size assigned for remediations' os data (in number of elements).       |
++                                                    +----------------+----------------------------------------------------------------------------------+
```
Comment (rauldpm) [inline source/user-manual/reference/internal-options.rst:1011]: `lru` is an acronym, it should be `LRU`

File: source/user-manual/reference/internal-options.rst [hunk 3]
```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
...
+|                                                    | default value  | 2048                                                                             |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | any integer between 1 and 100000                                                 |
```
Comment (rauldpm) [inline source/user-manual/reference/internal-options.rst:1014]: `description`, `default`, `any`, and `allowed` should be capitalized, please check the rest of the table and follow the same style

**Review by rauldpm — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/internal-options.rst (hunk 2 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/internal-options.rst]: Done. Thanks

File: source/user-manual/reference/internal-options.rst (hunk 3 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/internal-options.rst]: Done. Thanks

File: source/user-manual/reference/internal-options.rst [hunk 4]
```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
...
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for remediations' os data (in number of elements).       |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | Default value  | 2048                                                                             |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | Any integer between 1 and 100000                                                 |
```
Comment (rauldpm) [inline source/user-manual/reference/internal-options.rst:1014]:
```suggestion
|                                                    | Allowed values | Any integer between 1 and 100000                                                 |
```

**Review by rauldpm — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/internal-options.rst (hunk 4 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/internal-options.rst]: Done!

File: source/user-manual/reference/internal-options.rst [hunk 5]
```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for remediations' os data (in number of elements).       |
```
Comment (javimed) [inline source/user-manual/reference/internal-options.rst:1010]:
```suggestion
|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for vulnerability remediation (in number of elements).   |
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/internal-options.rst (hunk 5 above)
Comment (javimed) [reply source/user-manual/reference/internal-options.rst:1010]: (Unless `remediations' os data` meant something else.)

Comment (sebasfalcone) [reply source/user-manual/reference/internal-options.rst:1010]: No, makes sense to change it

File: source/user-manual/reference/internal-options.rst [hunk 6]
```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for vulnerability remediation (in number of elements).   |
```
Comment (sebasfalcone) [inline source/user-manual/reference/internal-options.rst:1010]:
```suggestion
|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for vulnerability remediations (in number of elements).   |
```

**Review by sebasfalcone — APPROVED** [review]

**Review by javimed — APPROVED** [review]

**Review by rauldpm — APPROVED** [review]

**Pull request merged**